package.json

```diff
@@ -15,7 +15,7 @@
     "git-info": "rm -rf src/generated/ && mkdir src/generated/ && echo export default \"{\\\"commitHash\\\": \\\"$(git rev-parse --short HEAD)\\\", \\\"version\\\": \\\"$(git describe --tags --always)\\\"};\" > src/generated/gitInfo.ts"
   },
   "dependencies": {
-    "@stellar/design-system": "^2.0.0-beta.5",
+    "@stellar/design-system": "^2.0.0-beta.6",
```

Comment:
should this be `beta.7`

Reply:
No, this PR doesn't need `v7`. The parent branch will have it updated.